modules/nigiri/src/routing.cc

```diff
@@ -155,6 +157,13 @@ motis::module::msg_ptr route(tag_lookup const& tags, n::timetable const& tt,
   auto extend_interval_later = false;
   auto start_time = n::routing::start_time_t{};
   auto start_station = n::location_idx_t::invalid();
+  auto timeout = [&]() -> std::optional<std::chrono::seconds> {
+    if (req->timeout() == 0) {
+      return {};
```

Comment:
`std::nullopt` to make it more obvious, what's happening here.

```suggestion
      return std::nullopt;
```